### DISCUSSION TITLE
not found @openzeppelin/contracts/token/ERC271/ERC271.sol

### DISCUSSION THREAD

**Discussion by ENGNFT** [smartcontractkit/full-blockchain-solidity-course-py#1448] [Q&A]
i tired compilying the contract that message showed  error not found @openzeppelin/contracts/token/ERC271/ERC271.sol   in remix.ethereum website how to solve this issue what code to import 
![error](https://user-images.githubusercontent.com/104599436/165855576-05434171-b179-41e7-83b0-e5e6c5a506de.png)

**Comment by cromewar** [comment]
Hello @ENGNFT 
We will need more info on this, please give a check to #980 ;)
Anyways I have some suggestions for you:

1. Try using double quotes: ""
2. Don't forget the contract identifier.

**Reply by ENGNFT** [reply]
can you tell me whats the contrat identifier ?    the error it says it doesnt find ERC271 still after puting " " 
![error2](https://user-images.githubusercontent.com/104599436/165864768-c6316d5f-5566-4451-bc78-4cad1773baae.png)

if there is error or update i should know about because i cant import it it isnt found

**Reply by cromewar** [reply]
In that case ve sure the npm package for openzeppelin is like the one you are trying to use, also sometimes remix freaks out and using another browser like Firefox should help.

**Comment by ENGNFT** [comment]
i downloaded and used firefox its the same issue i tried to code install npm said the identifier isnt found 
![error3](https://user-images.githubusercontent.com/104599436/165947372-15bfaa6c-6560-4019-92ed-15d4cb2fb3bc.png)
![errors4](https://user-images.githubusercontent.com/104599436/165947388-542172f6-2a18-4d41-925a-8f260e8274a0.png)

**Comment by cromewar** [comment]
Please give it a check to this: https://www.youtube.com/watch?v=c1Ws-Q5l-wg
